apps/ai/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@calcom/ai",
-  "version": "1.1.1",
+  "version": "1.2.0",
   "private": true,
   "author": "Cal.com Inc.",
   "dependencies": {
     "@calcom/prisma": "*",
     "@t3-oss/env-nextjs": "^0.6.1",
     "langchain": "^0.0.131",
     "mailparser": "^3.6.5",
-    "next": "^13.4.6",
+    "next": "^13.4.7",
```

Comment:
For now, we just minimally bumped to 13.4.7
Didn't commit yarn.lock so not sure how this will reconcile with the rest of the mono repo
Important to ref this PR which aims to bump whole mono to 13.5.4: https://github.com/calcom/cal.com/pull/11869

Reply:
I'm not sure, but we might not need to update nextjs across the whole mono repo. I think @baileypumfleet added a flag that lets the yarn.lock file get mutated in the build step in vercel, so it's possible the build will pass in vercel without changes to yarn.lock.

We could easily test this by approving the preview deployment of the AI app.

Deployment works for our internal Vercel deployment...

<img width="203" alt="Screenshot 2023-10-13 at 7 28 30 PM" src="https://github.com/calcom/cal.com/assets/36115192/639acfb6-36a8-4608-8716-be069c7f1bf6">


Could someone from Vercel team approve just the preview deployment of the AI app?
@PeerRich ?